src/V3Case.cpp

```diff
@@ -179,29 +180,50 @@ class CaseVisitor final : public AstNVisitor {
                     V3Number numval(itemp, iconstp->width());
                     numval.opBitsOne(iconstp->num());
                     uint32_t val = numval.toUInt();
-                    for (uint32_t i = 0; i < (1UL << m_caseWidth); ++i) {
+
+                    uint32_t first_overlap = ~(uint32_t)0;
```

Comment:
```suggestion
                    uint32_t first_overlap = ~static_cast<uint32_t>(0);
```
But reading code below I think it would be a lot clearer adding a new bool first_overlap_set = false; instead of using a magic value.

Reply:
Yeah, maybe. I couldn't decide whether this was clever and cute or confusing (or both). On balance, maybe it's just confusing - I'll switch to something explicit.